helm-charts/seldon-core-operator/templates/deployment_seldon-controller-manager.yaml

```diff
@@ -21,6 +21,9 @@ spec:
   template:
     metadata:
       annotations:
+{{- with .Values.manager.annotations }}
+{{- toYaml . | nindent 4}}
```

Comment:
:question: Couple of days late, but shouldn't this be an indent 8 spaces not 4?

Reply:
yep--i'll fix in another pr. Good catch!